go/parquet/pqarrow/schema.go

```diff
@@ -32,6 +32,10 @@ import (
 	"golang.org/x/xerrors"
 )
 
+// constants for the extension type metadata keys for the type name and
+// any extension metadata to be passed to deserialize.
+const	ExtensionTypeKeyName = "ARROW:extension:name"
+
```

Comment:
you should just use `const ExtensionTypeKeyName = ipc.ExtensionTypeKeyName` or just use `ipc.ExtensionTypeKeyName` directly since it's already exported there. No need to hardcode it here.